src/v/utils/log_hist.cc

```diff
@@ -11,13 +11,10 @@
 
 #include "utils/log_hist.h"
 
-template<
-  typename duration_t,
-  int number_of_buckets,
-  uint64_t first_bucket_upper_bound>
+template<int number_of_buckets, uint64_t first_bucket_upper_bound>
```

Comment:
you can skip reviewing this commit, I made a separate PR out of it https://github.com/redpanda-data/redpanda/pull/21451